metadata-ingestion/src/datahub/ingestion/source/metadata/lineage.py

```diff
@@ -0,0 +1,179 @@
+import logging
+from dataclasses import dataclass, field
+from typing import Iterable, List, Optional, Union
+
+from pydantic import validator
+
+import datahub.metadata.schema_classes as models
+from datahub.cli.cli_utils import get_aspects_for_entity
+from datahub.configuration.common import ConfigModel
+from datahub.configuration.config_loader import load_config_file
+from datahub.emitter.mce_builder import (
+    get_sys_time,
+    make_dataset_urn_with_platform_instance,
+)
+from datahub.emitter.mcp import MetadataChangeProposalWrapper
+from datahub.ingestion.api.source import Source, SourceReport
+from datahub.ingestion.api.workunit import MetadataWorkUnit, UsageStatsWorkUnit
+
+logger = logging.getLogger(__name__)
+
+auditStamp = models.AuditStampClass(
+    time=get_sys_time(), actor="urn:li:corpUser:pythonEmitter"
+)
+
+
+class EntityConfig(ConfigModel):
+    name: str
+    type: str
```

Comment:
I decided against it to let the recipe continue for other valid entity nodes / configurations rather than raising an error for it. (see lines 136-140 and 151-155). Although, I guess we could let the validator raise this warning message instead. Let me know what you think.

https://github.com/linkedin/datahub/blob/a48884340263af12de9088c3fba079ed3fabfde9/metadata-ingestion/src/datahub/ingestion/source/metadata/lineage.py#L136-L140

https://github.com/linkedin/datahub/blob/a48884340263af12de9088c3fba079ed3fabfde9/metadata-ingestion/src/datahub/ingestion/source/metadata/lineage.py#L151-L155

Reply:
Sounds good. Will add.